src/services/car/messageLead.ts

```diff
@@ -245,3 +246,75 @@ export const resendMessageLead = async ({
     result: {},
   }
 }
+
+export const fetchDealerWhatsappLeads = async ({
+  dealerId,
+  query,
+  options = {},
+}: {
+  dealerId: number
+  query: LeadQueryParams
+  options?: ApiCallOptions & { validateOnly?: boolean }
+}): Promise<Paginated<SearchWhatsappLead>> => {
+  const { validateOnly, ...otherOptions } = options
+  const { page, size, sort = {}, searchQuery } = query
+
+  const { sortOrder, sortType } = sort
+
+  const sortOrDefault = {
+    sortType: sortType || defaultLeadSort.sortType,
+    sortOrder: sortOrder || defaultLeadSort.sortOrder,
+  }
+
+  const queryParams = {
+    page: pageOrDefault(page, defaultDealerMessageLeadsPagination),
```

Comment:
Observation: `defaultDealerMessageLeadsPagination` is used in `call`, `message` and  now `whatsapp` leads. Let's rename it to `defaultLeadsPagination` to keep it in line with default options for sort